src/pydata_sphinx_theme/assets/scripts/pydata-sphinx-theme.js

```diff
@@ -473,7 +473,9 @@ function showVersionWarningBanner(data) {
     return;
   }
   // now construct the warning banner
-  var outer = document.createElement("div");
+  var outer = document.createElement("aside");
+  // TODO: add to translatable strings
```

Comment:
I want to do this in a follow-up PR not this one because this PR has been open for a while and has gotten stuck on translation stuff, so I would like to tackle the translation issues separately.

Reply:
The string here, "Version warning", is used to label the landmark (`<aside>` => role = complementary) that I added to the version warning banner. (The rationale is that since we make the version warning banner so visually prominent, we should also make it prominent for assistive tech, hence the landmark.) 

Eventually the string needs to be translated. It will be surfaced by assistive tech when showing or announcing landmarks on the page: "Version warning, complementary" (something like that)

Reply:
Can you open a new issue to remind us to come back to this (question about marking strings as translatable if they're generated within JavaScript) please?